<launch>
    <node name="line_extractor_node" pkg="sb_pointcloud_processing" type="line_extractor_node" output="screen">
        <!-- degree of polynomial for line of best fit -->
        <param name="degree_polynomial" value="3" type="int" />
        <!-- regularization constant for line of best fit -->
        <param name="lambda" value="0" type="double" />
        <!-- density parameters for DBSCAN -->
        <param name="min_neighbours" value="3" type="int" />
        <param name="radius" value="5" type="double" />
    </node>
    <node name="test_pcl_generator_node" pkg="sb_pointcloud_processing" type="test_pcl_generator_node" output="screen">
        <rosparam param="first_line">[50, 0, -0.01]</rosparam>
        <rosparam param="second_line">[0, 0, -0.01]</rosparam>
        <param name="x_min" value="-50" type="double" />
        <param name="x_max" value="50" type="double" />
        <param name="x_delta" value="0.5" type="double" />
        <param name="max_noise_x" value="5" type="double" />
        <param name="max_noise_y" value="5" type="double" />
    </node>
</launch>